docs/source/checks/vision/train_test_validation/plot_image_dataset_drift.py

```diff
@@ -116,3 +117,14 @@ def batch_to_images(self, batch):
 # ^^^^^^^^^^^^^^^^^^^
 check = ImageDatasetDrift()
 check.run(train_dataset=drifted_train_ds, test_dataset=test_ds)
+
+
+#%%
+# Define a Condition
+# -------------
+# Now, we will define a condition that the maximum drift score is less than a certain threshold. In this example we will
+# set the threshold at 0.8.
+
+check.add_condition_drift_score_less_than(0.8)
```

Comment:
this is a lot of drift,
why not setting it to a more common threshold?

Reply:
Because the drift score in this case is 0.709 and I want it to pass